src/materials/pathtracing/PhysicalPathTracingMaterial.js

```diff
@@ -361,6 +361,7 @@ export class PhysicalPathTracingMaterial extends MaterialBase {
 									// NOTE: Only area lights are supported for forward sampling and can be hit
 									float misWeight = misHeuristic( scatterRec.pdf, lightRec.pdf / lightsDenom );
 									gl_FragColor.rgb += lightRec.emission * state.throughputColor * misWeight;
+									gl_FragColor.a = lightRec.visibleSurface; // This is not working as expected
```

Comment:
This section of code only runs when the current ray is _not_ a camera ray (a ray representing light that goes directly into the camera - not reflected) - see the condition above:

```c
if ( ! state.firstRay && ! state.transmissiveRay ) {
```

Conversely we only want to display the light surface and exit if the current ray _is_ a camera ray. So we'll need to add a new loop to find the light surface in an else statement:

```c
if ( ! state.firstRay && ! state.transmissiveRay ) {

    // accumulate light from direct lights hits for reflected rays

} else {

    // find the closest light with a visible surface, exit path tracing

}
```

Hopefully that makes sense?